browser/lib/util/http.js

```diff
@@ -3,6 +3,16 @@ var Http = (function() {
 
 	function Http() {}
 
+	function shouldFallback(err) {
+		var statusCode = err.statusCode;
+		/* 400 + no code = a generic xhr onerror. Browser doesn't give us enough
+		 * detail to know whether it's fallback-fixable, but it may be (eg if a
+		 * network issue), so try just in case */
+		return (statusCode === 408 && !err.code) ||
+			(statusCode === 400 && !err.code)      ||
```

Comment:
That's what I meant by 'kludge it with spaces' (and what https://github.com/ably/ably-js/commit/68cf9ca1896015240e3db375566ccabd10e8f8f5 did), sorry if that wasn't clear.

IMHO it still a kludge because you end up doing
```
<tab>return foo ||
<tab>  bar      ||
```
only when you do want to align ||'s; if not, then you leave it at what your editor does automatically, ie
```
<tab>return foo ||
<tab><tab>bar;
```
so the beginning-of-line whitespace on the remaining lines changes depending on whether there's something you want to align at the end of the line, hence kludgy